events/equeue/equeue_mbed.cpp

```diff
@@ -131,7 +131,7 @@ static void equeue_sema_timeout(equeue_sema_t *s) {
 bool equeue_sema_wait(equeue_sema_t *s, int ms) {
     int signal = 0;
     Timeout timeout;
-    if (ms > 0) {
+    if (ms >= 0) {
```

Comment:
@pan-, should be updated. At the very least this makes the less travelled code path clearer.